windows/RNScreens/RNScreens.vcxproj

```diff
@@ -15,12 +15,12 @@
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
   </PropertyGroup>
   <PropertyGroup Label="ReactNativeWindowsProps">
-    <ReactNativeWindowsDir Condition="'$(ReactNativeWindowsDir)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'node_modules\react-native-windows\package.json'))\node_modules\react-native-windows\</ReactNativeWindowsDir>
+    <ReactNativeWindowsDir Condition="'$(ReactNativeWindowsDir)' == ''">$([MSBuild]::GetDirectoryNameOfFileAbove($(SolutionDir), 'node_modules\react-native-windows\package.json'))\node_modules\react-native-windows\</ReactNativeWindowsDir>

```

Comment:
Solution from react-native-windows repo: [https://github.com/microsoft/react-native-windows/issues/5911#issuecomment-730026225](https://github.com/microsoft/react-native-windows/issues/5911#issuecomment-730026225)
`the general fix for having an example app with its own pacakage.json/node_modules in the same repo as their module, and therefore ending up with two copies of react-native-windows - is to update the ReactNativeWindowsDir property in the module's project`